votes/initiateNewVote/_EDIT_ME.yml

```
@@ -35,10 +35,25 @@ canShuffleCandidates: true
 # 5. Insert here a short description of the vote objectives and link to the
 #    issue it was discussed on to give the full context.
 footerInstructions: |
-  TBD
+  The objective of this vote is to decide whether duration to determine
+  inactivity should be reduced from 12 months to 9 months. This vote was discussed
+  in pull-request https://github.com/nodejs/node/pull/52459 and the issue #1524.
 
 # 6. Optionally, insert a brief introduction for the vote PR, in the markdown format.
 prBody: |
+  Currently, the bot looks through all commits on `main` that have landed in the last 12 months, and makes two lists:
+  
+    - a list of all the commit authors.
+    - a list of all the collaborators who are listed as reviewers in the commit message (i.e. folks who have approved the PR). 
+    
+    The bot will open a PR to move a collaborator to emeritus if they don’t appear in either of those lists. 
```

```suggestion
  The bot will open a PR to move a collaborator to emeritus if they don’t appear in either of those lists. 
```

(Pretty sure it would look off with the extra indentention).